docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,443 @@
+
+   For example:
+   ```
+   fabric-ca-client enroll -d -u https://rcaadmin:rcaadminpw@my-machine.example.com:7055 --tls.certfiles ../crypto/ca-cert.pem --csr.hosts 'host1,*.example.com'
+   ```
+
+When this command completes, an `/msp` folder is generated under `/org1-ca/admin/msp` and contains the signing cert and private key for the org1 CA and looks similar to:
+
+```
+└── msp
+   ├── IssuerPublicKey
+   ├── IssuerRevocationPublicKey
+   ├── cacerts
+   │   └── my-machine-example-com-7055.pem
+   ├── keystore
+   │   └── 60b6a16b8b5ba3fc3113c522cce86a724d7eb92d6c3961cfd9afbd27bf11c37f_sk
+   ├── signcerts
+   │   └── cert.pem
+   └── user
+```
+
+Where:
+- `my-machine-example-com-7055.pem` is the **Organization CA root certificate**.
+- `60b6a16b8b5ba3fc3113c522cce86a724d7eb92d6c3961cfd9afbd27bf11c37f_sk` is the **private key** for the organization CA admin identity. This key needs to be protected and should not be shared with anyone. It is required to be able to register and enroll other identities with this CA.
+- `cert.pem` is the CA admin identity **signing certificate**.
+
+## (Optional) Deploy an intermediate CA
+
+Intermediate CAs form a chain a trust with the root CA and can be used to direct enrollment requests for a specific organization to a single CA as well as protect the root of trust by shutting down the root CA. When intermediate CAs are addressing all the enrollment requests, the root CA can be turned off.
+
+**Note:** This section assumes that you have already [registered and enrolled](optional-register-the-intermediate-ca-admin-with-the-tls-ca) the `icaadmin` identity with the TLS CA.
+
+### Before you start
+
+- Copy the Fabric CA server binary `fabric-ca-server` to a new directory on your machine.
+- Make sure you have copied the CA admin TLS certificate and key pair that you generated with the TLS CA for the icaadmin user to a location that can be accessed by this CA server. These are the `/msp/signcerts/cert.pem` and `/msp/keystore/` files that were generated by the enroll command in the /tls-ca folder.
+
+After deploying the root CA, you can perform these steps to deploy an intermediate CA:
+
+1. You already registered the `icaadmin` identity with the TLS CA. You need to also register the same identity with the (root) organization CA and because this will be an admin of an intermediate CA, you need to include the `hf.IntermediateCA=true` attribute.  (Run this command from the same terminal window where you enrolled the organization CA admin in the previous step.)
+   ```
+     fabric-ca-client register -u https://my-machine.example.com:7055  --id.name icaadmin --id.secret icaadminpw --id.attrs '"hf.Registrar.Roles=user,admin","hf.Revoker=true","hf.IntermediateCA=true"' --tls.certfiles ../crypto/ca-cert.pem
+   ```   
+2. From the intermediate CA home directory, initialize the CA by running the `init` command and bootstrapping the `icaadmin` id that you already registered with the TLS CA and the root CA. For example:
```

Comment:
The identity you register needs to be the username and password that are used by the intermediate CA. The intermediate ca enrolls itself automatically using what you provide on the init command. You actually need to pass to root ca username and password, which was not the order I was expecting. May be worth providing a higher level overview of that here.

Reply:
I clarified this point in the section where you register the int ca user with the root ca.